less/pages/encrypt/components.less

```diff
@@ -177,6 +177,7 @@ video{
   position: relative;
   margin: 5px;
   flex: 1 1 40%;
+  min-width: 0;
```

Comment:
Fortunately this seems to be fixed in upcoming builds, so I'm just gonna go with this since it's here and works. Thanks!